obelix/idefix/trigger/request.go

```diff
@@ -0,0 +1,16 @@
+package trigger
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+func ReadBody(res *http.Response) ([]byte, error) {
+	body, err := ioutil.ReadAll(res.Body)
+
```

Comment:
I'm not sure this function is really useful since you'll simply do in other function
```go
data, err := ReadBody(res)
if err != nil {
   // Handle error
}
```
You do not really win line and `ioutil.ReadAll(res.Body)` is explicit

Reply:
I don't understand